docs-chef-io/content/inspec/resources/host.md

```diff
@@ -11,100 +11,124 @@ platform = "os"
     parent = "inspec/resources/os"
 +++
 
-Use the `host` Chef InSpec audit resource to test the name used to refer to a specific host and its availability, including the Internet protocols and ports over which that host name should be available.
+Use the `host` Chef InSpec audit resource to test the specific host name and its availability. This test includes the internet protocols and ports on which the respective host name must be available.
 
 ## Availability
 
 ### Installation
 
-This resource is distributed along with Chef InSpec itself. You can use it automatically.
+The Chef InSpec distributes this resource.
 
 ### Version
 
-This resource first became available in v1.0.0 of InSpec.
+This resource is available from InSpec version 1.0.
 
 ## Syntax
 
-A `host` resource block declares a host name, and then (depending on what is to be tested) a port and/or a protocol:
+A `host` resource block declares a host name, a port, and a protocol.
 
     describe host('example.com', port: 80, protocol: 'tcp') do
       it { should be_reachable }
       it { should be_resolvable }
       its('ipaddress') { should include '12.34.56.78' }
     end
 
-where
-
-- `host()` must specify a host name and may specify a port number and/or a protocol
-- `'example.com'` is the host name
-- `port:` is the port number
-- `protocol:` is the Internet protocol: TCP (`protocol: 'tcp'`), UDP (`protocol: 'udp'` or ICMP (`protocol: 'icmp'`))
+> where
+>
+> - `host()` must specify a host name, a port number, and a protocol.
```

Comment:
port number and protocol are optional arguments whereas the statement host() must specify a host name, a port number, and a protocol. makes it sound all are mandatory.

Reply:
ok. Please check and confirm

Reply:
Looks good